### ISSUE TITLE
Avatar keeps running without input from the player

### ISSUE THREAD

**Issue by olavra** [decentraland/unity-renderer#1900]
Several player have reported their avatar moves without pushing any key.

https://github.com/decentraland/issues/issues/282
https://github.com/decentraland/issues/issues/287
https://github.com/decentraland/issues/issues/251
https://github.com/decentraland/issues/issues/210
https://github.com/decentraland/issues/issues/187
https://github.com/decentraland/issues/issues/180
https://github.com/decentraland/issues/issues/173

**Acceptance Criteria**
Avatar should not move without player input

**Comment by JorgeDecentraland** [comment]
@davidejensen please validate with users that this is not happening anymore

**Comment by davidejensen** [comment]
The estimate may vary a lot based on the user feedback

**Comment by davidejensen** [comment]
Replied to all the cited issues. Other than 2 the others were unanswered from the original poster for a long time.
The issue hasn't been solved for everyone because there are some reports after the PR has been merged.
One hint that I got from an issue is that the problem could be related to laptops with touchscreen support or touchpen support but I haven't got one, nor I have ever tried it. I asked for some additional details/tests from the user.
